scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime, timedelta
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+m = DuplicateModel.load("duplicatemodel")
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+test_bugs = bugzilla.download_bugs_between(
+    datetime.now() - timedelta(days=21), datetime.now(), store=False
+)
```

Comment:
How long does the download take? If it takes a long time, you could store the data in a json file

Reply:
It takes about 6-7 minutes.

Reply:
OK, so maybe it's worth doing that so that you can iterate faster.
Just do something like:
```
try:
    with open('duplicate_test_bugs.json', 'r') as f:
        test_bugs = json.load(f)
except FileNotFoundError:
        test_bugs = DOWNLOAD
        with open('duplicate_test_bugs.json', 'w') as f:
            json.dump(test_bugs, f)
```

Reply:
Sure, also, I get an unexpected error while classifying the model :
```
from bugbug import bugzilla
from bugbug.models.duplicate import DuplicateModel as dm
bugs = bugzilla.get_bugs()
b=  (next(bugs), next(bugs))
m = dm.load("duplicatemodel")

m.classify(b)
```

Error : 
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ayush99/GitHub/bugbug/bugbug/model.py", line 191, in classify
    X = self.extraction_pipeline.transform(items)
  File "/home/ayush99/GitHub/bbug/lib/python3.7/site-packages/sklearn/pipeline.py", line 540, in _transform
    Xt = transform.transform(Xt)
  File "/home/ayush99/GitHub/bugbug/bugbug/bug_features.py", line 444, in transform
    author_ids = get_author_ids() if self.commit_map else None
AttributeError: 'BugExtractor' object has no attribute 'commit_map'
```
I wasn't getting this earlier today,  not sure what's causing this. @marco can you confirm this?